development/cleanup.sh

```diff
@@ -3,7 +3,7 @@
 #goto git root directory
 git rev-parse && cd "$(git rev-parse --show-cdup)"
 
-. development/env.sh
+#. development/env.sh
```

Comment:
Seems like some extra changes sneaked to the PR

Reply:
Ooops :) yeah my local change.